src/core/View.js

```diff
@@ -511,6 +511,9 @@ export default class View extends Component {
       return this.selections.map((v) => {
         const { prop, compositeID, displayPosition } = v.getProperties();
 
+        // Return false to mark this item for removal
+        if (prop == null) return false;
```

Comment:
Should we write `if (!prop) { return false; }` instead?

Reply:
I'm used to `== null` because it's the quickest way to check for `null` and `undefined` while not checking for any other falsy value, I know in this case it will always be either an object-type or null/undefined so it's not strictly needed, i't just clearer to me. (I come from a strict Flow-type checking background, sorry 😬)